src/main/java/org/icatproject/utils/IcatUnits.java

```diff
@@ -0,0 +1,89 @@
+package org.icatproject.utils;
+
+import javax.measure.IncommensurableException;
+import javax.measure.UnconvertibleException;
+import javax.measure.Unit;
+import javax.measure.UnitConverter;
+import javax.measure.format.MeasurementParseException;
+
+import tech.units.indriya.format.SimpleUnitFormat;
+
+/**
+ * Utility to perform conversions to SI (System) units.
+ */
+public class IcatUnits {
+
+	/**
+	 * Holds the SI units and value for a quantity. If the units provided at
+	 * construction could not be parsed, then these will be null.
+	 */
+	public class SystemValue {
+		public String units = null;
+		public Double value = null;
+
+		/**
+		 * Converts value units into an SI quantity.
+		 * 
+		 * @param value Quantity in (potentially) non-SI units.
+		 * @param units Units of the provided value.
+		 */
+		public SystemValue(Double value, String units) {
```

Comment:
While it could because `unitFormat` is static, `unitFormat` probably shouldn't be static in the first place. We don't provide the aliases at the creation of the `SimpleUnitFormat`, but do this in the `IcatUnits` constructor(s). So while we can statically create the `SystemValue`, whether the units get converted or not will depend on the constructor called for `IcatUnits`, which is not static behaviour.

I'd be tempted to redo this more fundamentally:
- Make `unitFormat` non-static
- Move `SystemValue` to its own file
- Refactor the constructor of `SystemValue` into a non-static function on `IcatUnits` (e.g. `createSystemUnit`) that returns a `SystemUnit`, removing the need to access `unitFormat` in `SystemValue` at all.

This would probably close to how other simple representations of search results, facets etc. have been done elsewhere:
https://github.com/icatproject/icat.server/blob/267_opensearch_support/src/main/java/org/icatproject/core/manager/search/SearchResult.java
https://github.com/icatproject/icat.lucene/blob/18_search_improvements/src/main/java/org/icatproject/lucene/FacetedDimension.java

What do you think?